### PR TITLE
[SQL Exercises] Updated SQLFiddle links, rephrased some questions, updated link types, and more.

### PR DESCRIPTION
**Pull request by milesflo**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-alter-table-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-alter-table.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-define-default-seq-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-define-default-seq-3.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-define-default-seq-4.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/sql-practice-define-default-seq.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-create-table-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-create-table-3.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-create-table-4.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-create-table.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-define-relationships-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-define-relationships-3.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-define-relationships-4.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/sql-practice-define-relationships.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-numeric-data-types.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-string-data-types.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-text-data-type.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/create-indexes.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/sql-practice-define-indexes.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/sql-practice-sql-indexes2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/when-to-add-indexes.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-mssql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-mysql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-psql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-an-rdbms.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-mssql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-mysql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-postgresql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/avg-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/count-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/distinct-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/group-by-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/min-and-max-clauses.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-average.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-group.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-single-table-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-single-table.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-table-max-filter.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sql-practice-aggregate-table-min-and-max.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sum-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/order-by-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/select.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/sql-practice-order-results.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/sql-practice-select.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/sql-practice-string-matching.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-3.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/select-all.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/select-cols.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/select-where.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/sql-practice-select-1.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/sql-practice-select-2.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-selects/sql-practice-where-comparison.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/inner-join.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/multiple-joins.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/sql-practice-average-aggregate-multiple-tables.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/sql-practice-join-multiple-tables.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/sql-practice-max-aggregate-multiple-tables.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/sql-practice-min-max-multiple-tables.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/database-server-types.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/what-is-a-database-server.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/what-is-an-rdbms.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/what-is-sql.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/delete-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/sql-practice-delete.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/sql-practice-insert.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/sql-practice-update.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/update-clause.md?ref=149ef08ba8e61c4ddf094a02899fcefe3c9d1d68

File: sql/dql/basic-queries/sql-practice-order-results.md [hunk 1]
```diff
@@ -23,11 +23,12 @@ standards:
   sql.read-single-table.3: 1000
 
 links:
-  - '[MySQL ORDER BY documentation](https://dev.mysql.com/doc/refman/5.7/en/order-by-optimization.html){website}'
+  - '[MySQL ORDER BY documentation](https://dev.mysql.com/doc/refman/5.7/en/order-by-optimization.html){documentation}'
   - '[blog post tutorial on SELECT](http://www.dofactory.com/sql/select){website}'
   - '[video tutorial on MySQL SELECT](https://www.youtube.com/watch?v=BgK88mlgA6I){video}'
 
 ---
+
```
Comment (Loopiezlol) [inline sql/dql/basic-queries/sql-practice-order-results.md]: newlines above section name are not needed btw
imo it looks better without them

File: sql/dql/write/update-clause.md [hunk 2]
```diff
@@ -88,12 +88,12 @@ id |     name
...
 
 Note that if the `WHERE` clause isn't used all the values in the specified column will be updated:
-```
+```sql
 UPDATE egg_group
 SET name = 'rabbit';
 ```
 *Result:*
-```sql
+```
```
Comment (Loopiezlol) [inline sql/dql/write/update-clause.md]: you can put 'txt' for plain text

File: sql/ddl/alter/sql-practice-alter-table-2.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ standards:
 
 
 links:
-  - '[MySQL modify columns documentation](https://dev.mysql.com/doc/refman/5.7/en/alter-table.html){website}'
+  - '[MySQL modify columns documentation](https://dev.mysql.com/doc/refman/5.7/en/alter-table.html){Documentation}'
```
Comment (Loopiezlol) [inline sql/ddl/alter/sql-practice-alter-table-2.md]: probably we want it with lowercase, not sure it's automatically done before going in the db

**Pull request merged**